### PR TITLE
README.md: add build steps for Visual Studio 2019 / MSVC

### PR DESCRIPTION
**Pull request by greg7mdp**

### CODE REVIEW

**Comment by greg7mdp** [comment]
I just validated that the above steps indeed build material_sandbox successfully, and I was able to run it.

File: README.md [hunk 1]
```diff
@@ -341,6 +341,33 @@ See [ios/samples/README.md](./ios/samples/README.md) for more information.
 
 ### Windows
 
+#### Building on Windows with the Visual Studio 2019 compiler
+
+Install the following components:
+
+- [Visual Studio 2019](https://www.visualstudio.com/downloads)
+- [Python 3.7](https://www.python.org/ftp/python/3.7.0/python-3.7.0.exe)
+- [Cmake 3.13 or later](https://github.com/Kitware/CMake/releases/download/v3.13.4/cmake-3.13.4-win64-x64.msi)
```
Comment (bejado) [inline README.md]: I think CMake 3.14 is the first one to have the Visual Studio 2019 generator.

Comment (bejado) [reply README.md]: Also, Cmake -> CMake

File: README.md [hunk 2]
```diff
@@ -341,6 +341,33 @@ See [ios/samples/README.md](./ios/samples/README.md) for more information.
...
+
+Open an appropriate Native Tools terminal for Visual Studio, for example the `Developer Command Prompt for VS2019`.
```
Comment (bejado) [inline README.md]: Please mention that it needs to be a x64 terminal.

File: README.md [hunk 3]
```diff
@@ -341,6 +341,33 @@ See [ios/samples/README.md](./ios/samples/README.md) for more information.
...
+> mkdir out
+> cd out
+> cmake ..
+```
+
+Then, you should be able to load the generated solution file `TNT.sln` in Visual Studio and build the `material_sandbox` project.
+
+Run it from the `out` directory with:
+```
+> ssamples/Debug/material_sandbox.exe ../assets/models/monkey/monkey.obj
```
Comment (bejado) [inline README.md]: ssamples -> samples

File: README.md (hunk 1 above)
Comment (greg7mdp) [reply README.md]: thanks, done.

File: README.md (hunk 2 above)
Comment (greg7mdp) [reply README.md]: thanks, done.

**Comment by greg7mdp** [comment]
I noticed that the INSTALL project does not discriminate between the debug and release builds... always copy into the `install` directory. Would it be difficult to append the CMAKE_INSTALL_CONFIG_NAME at the end, so we would install to `install-Debug` or `install-Release`?

**Comment by romainguy** [comment]
I'd rather avoid this as it would impact `build.sh` and various CI scripts. I don't know what Visual Studio does but our build script for other operating systems does it the CMake way:

```-DCMAKE_INSTALL_PREFIX=../${lc_target}/filament \```

i.e. we choose a different install prefix per build type.

**Comment by greg7mdp** [comment]
@romainguy ok that's fine, this was just a convenience for people using the VS2019 environment.

File: README.md [hunk 4]
```diff
@@ -341,6 +341,33 @@ See [ios/samples/README.md](./ios/samples/README.md) for more information.
...
+> mkdir out
+> cd out
+> cmake ..
+```
+
+Then, you should be able to load the generated solution file `TNT.sln` in Visual Studio and build the `material_sandbox` project.
+
+Run it from the `out` directory with:
+```
+> samples/Debug/material_sandbox.exe ../assets/models/monkey/monkey.obj
```
Comment (bejado) [inline README.md]: Please use Windows-style backslashes: \

File: README.md [hunk 5]
```diff
@@ -341,6 +341,33 @@ See [ios/samples/README.md](./ios/samples/README.md) for more information.
 
 ### Windows
 
+#### Building on Windows with the Visual Studio 2019 compiler
+
+Install the following components:
+
+- [Visual Studio 2019](https://www.visualstudio.com/downloads)
+- [Python 3.7](https://www.python.org/ftp/python/3.7.0/python-3.7.0.exe)
+- [CMake 3.14 or later](https://github.com/Kitware/CMake/releases/download/v3.13.4/cmake-3.13.4-win64-x64.msi)
```
Comment (bejado) [inline README.md]: Please fix the link to CMake 3.14

File: README.md [hunk 6]
```diff
@@ -341,6 +341,33 @@ See [ios/samples/README.md](./ios/samples/README.md) for more information.
...
+
+Open an appropriate Native Tools terminal for Visual Studio for x64, for example the `Developer Command Prompt for VS2019`.
```
Comment (bejado) [inline README.md]:
```suggestion
Open the `x64 Native Tools Command Prompt for VS 2019`.
```

Comment (greg7mdp) [reply README.md]: thanks and done

**Pull request merged**